.vscode/launch.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
Please add `.vscode/` to the gitignore file.

Reply:
Done